util/kustomize/kustomize.go

```diff
@@ -110,6 +110,25 @@ func (k *kustomize) Build(opts *v1alpha1.ApplicationSourceKustomize) ([]*unstruc
 				}
 			}
 		}
+
+		if len(opts.CommonLabels) > 0 {
+			//  edit add label foo:bar
+			args := []string{"edit", "add", "label"}
```

Comment:
Does this work in both Kustomize versions 1 and 2?

Reply:
Well, it should... test covers only v2 :see_no_evil: k1 also offers this command with the exact same help text...

Reply:
Well, it should... test covers only v2 :see_no_evil: k1 also offers this command with the exact same help text...